scio-bigtable/src/main/scala/com/spotify/scio/bigtable/package.scala

```diff
@@ -80,11 +81,13 @@ package object bigtable {
       if (self.isTest) {
         // No need to update the number of nodes in a test
       } else {
-        BigtableUtil.updateNumberOfBigtableNodes(
```

Comment:
@rgruener can you wrap `BigtableClusterUtilities` in a `object BigtableUtil` to maintain backwards compatibility?